spec.html

```diff
@@ -16277,179 +16277,48 @@ <h2>Syntax</h2>
     <h1>White Space</h1>
     <p>White space code points are used to improve source text readability and to separate tokens (indivisible lexical units) from each other, but are otherwise insignificant. White space code points may occur between any two tokens and at the start or end of input. White space code points may occur within a |StringLiteral|, a |RegularExpressionLiteral|, a |Template|, or a |TemplateSubstitutionTail| where they are considered significant code points forming part of a literal value. They may also occur within a |Comment|, but cannot appear within any other kind of token.</p>
-    <p>The ECMAScript white space code points are listed in <emu-xref href="#table-white-space-code-points"></emu-xref>.</p>
-    <emu-table id="table-white-space-code-points" caption="White Space Code Points" oldids="table-32">
-      <table>
-        <tr>
-          <th>
-            Code Points
-          </th>
-          <th>
-            Name
-          </th>
-          <th>
-            Abbreviation
-          </th>
-        </tr>
-        <tr>
-          <td>
-            `U+0009`
-          </td>
-          <td>
-            CHARACTER TABULATION
-          </td>
-          <td>
-            &lt;TAB>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+000B`
-          </td>
-          <td>
-            LINE TABULATION
-          </td>
-          <td>
-            &lt;VT>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+000C`
-          </td>
-          <td>
-            FORM FEED (FF)
-          </td>
-          <td>
-            &lt;FF>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+FEFF`
-          </td>
-          <td>
-            ZERO WIDTH NO-BREAK SPACE
-          </td>
-          <td>
-            &lt;ZWNBSP>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            any code point in general category “Space_Separator”
-          </td>
-          <td>
-          </td>
-          <td>
-            &lt;USP>
-          </td>
-        </tr>
-      </table>
-    </emu-table>
-    <emu-note>
-      <p>U+0020 (SPACE) and U+00A0 (NO-BREAK SPACE) code points are part of &lt;USP>.</p>
-    </emu-note>
-    <emu-note>
-      <p>Other than for the code points listed in <emu-xref href="#table-white-space-code-points"></emu-xref>, ECMAScript |WhiteSpace| intentionally excludes all code points that have the Unicode “White_Space” property but which are not classified in general category “Space_Separator” (“Zs”).</p>
-    </emu-note>
     <h2>Syntax</h2>
     <emu-grammar type="definition">
       WhiteSpace ::
-        &lt;TAB&gt;
-        &lt;VT&gt;
-        &lt;FF&gt;
-        &lt;ZWNBSP&gt;
-        &lt;USP&gt;
+        &lt;U+0009 (CHARACTER TABULATION)&gt;
+        &lt;U+000B (LINE TABULATION)&gt;
+        &lt;U+000C (FORM FEED)&gt;
+        &lt;U+0020 (SPACE)&gt;
+        &lt;U+00A0 (NO-BREAK SPACE)&gt;
```

Comment:
```suggestion
```
Adding the SPACE and NO-BREAK SPACE code points into the WhiteSpace production reverses PR #2559, and seems out of scope for this PR.

Reply:
This was meant to simplify the added note below, which itself was meant to address #3303. But I can split it out into a follow-up PR.